### PR TITLE
ci: add unified CI to magento2

### PR DESCRIPTION
**Pull request by ghost**
[IN-3046](https://vsf.atlassian.net/browse/IN-3046)

- [ ] This is a public project. Need to get a release token later once it's merged

[IN-3046]: https://vsf.atlassian.net/browse/IN-3046?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -10,6 +10,29 @@ concurrency:
 
 jobs:
   continous-integration:
-    name: "Continuous Integration"
-    uses: vuestorefront/integrations-github-workflows/.github/workflows/continuous-integration.yml@main
+    name: 'Continuous Integration'
+    #                                                                                               \/ FIXME Change to develop once/if it's merged
```
Comment (Fifciu) [inline .github/workflows/continuous-integration.yml:14]: What's that?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,69 +0,0 @@
-# Contributing
```
Comment (Fifciu) [inline CONTRIBUTING.md:1]: Why did we remove it?

**Review by Fifciu — APPROVED** [review]

File: packages/api-client/possible-types.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
-/* eslint-disable unicorn/prefer-module */
-require('dotenv').config();
-const fetch = require('cross-fetch');
-const fs = require('fs');
+/* eslint-disable @typescript-eslint/no-var-requires */
```
Comment (WojtekTheWebDev) [inline packages/api-client/possible-types.js:1]: Can you add this rule as warn to new eslint confg?

**Review by michaelKurowski — DISMISSED** [review]

File: .changeset/config.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
+  "changelog": ["@changesets/changelog-github", { "repo": "vuestorefront/magento2" }],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
```
Comment (WojtekTheWebDev) [inline .changeset/config.json:9]:
```suggestion
  "updateInternalDependencies": "minor",
```

**Review by WojtekTheWebDev — DISMISSED** [review]
Only 1 suggestion

**Review by michaelKurowski — APPROVED** [review]

**Pull request merged**